protractor.conf.js

```diff
@@ -14,20 +29,52 @@ module.exports.config = {
         customTimeout: 3000,
         customMinTimeout: 500,
     },
-     /**
-     * How long to wait for a page to load.
-     */
+
+    // The timeout in milliseconds for each script run on the browser.
+    allScriptsTimeout: 60000,
```

Comment:
Reduce timeout to 20-30 secs - Waiting for Angular - http://www.protractortest.org/#/timeouts